spec/models/invoice_spec.rb

```diff
@@ -9,8 +9,8 @@
     it { should have_many(:merchants).through(:items) }
   end
   describe "class methods" do
-    describe "#find_incomplete_invoices" do
-      it "returns all the invoices that have items that have not been shipped yet" do
+    describe "#find_and_sort_incomplete_invoices" do
+      it "returns all the invoices that have items that have not been shipped yet, I see the creation date ordered by oldest to newest" do
```

Comment:
This is a little lengthy too.
```suggestion
      it "returns all invoices with unshipped items, ordered by creation, oldest to newest" do
```

Reply:
Love it x2, Updated!